build.sbt

```diff
@@ -1,16 +1,24 @@
 import sbtrelease.ReleasePlugin.autoImport.ReleaseTransformations._
 
-organization in ThisBuild := "ai.lum"
+ThisBuild / organization := "ai.lum"
 
-scalaVersion in ThisBuild := "2.12.10"
+ThisBuild / scalaVersion := "2.12.10"
```

Comment:
Should we make it a policy to use the highest _._.X version available, like 2.12.15, so that we get the most bug fixes?